build/DependencyVersions.props

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <MicrosoftAspNetCoreAllPackageVersion>2.2.6</MicrosoftAspNetCoreAllPackageVersion>
+    <MicrosoftAspNetCoreAllPackageVersion>2.2.7</MicrosoftAspNetCoreAllPackageVersion>
```

Comment:
This is already 2.2.7 in release/2.2.2xx  AFAICT 

https://github.com/dotnet/cli/blob/release/2.2.2xx/build/DependencyVersions.props

GitHub seems to be misleading me with the diff here.

Reply:
Pulling this down locally to verify

Reply:
This is the complete diff according `git diff release/2.2.2xx`

``` diff
diff --git a/build/Signing.proj b/build/Signing.proj
index 2c68da60e..fa00e13da 100644
--- a/build/Signing.proj
+++ b/build/Signing.proj
@@ -5,7 +5,7 @@
     <!-- The signing infrastructure runs using MSBuild 14, which doesn't support some of the new syntax we're using.  So set the BuildingSingingProject
          property here to avoid importing files we don't need for signing which would cause errors if imported when using MSBuild 14. -->
     <BuildingSigningProject>true</BuildingSigningProject>
-    <ExternalCertificateId Condition="'$(ExternalCertificateId)' == ''">135020001</ExternalCertificateId>
+    <ExternalCertificateId Condition="'$(ExternalCertificateId)' == ''">135020002</ExternalCertificateId>
     <InternalCertificateId Condition="'$(InternalCertificateId)' == ''">Microsoft402</InternalCertificateId>
     <NugetCertificateId Condition="'$(NugetCertificateId)' == ''">NuGet</NugetCertificateId>
   </PropertyGroup>
diff --git a/build/Version.props b/build/Version.props
index 51321876c..8bd2e6963 100644
--- a/build/Version.props
+++ b/build/Version.props
@@ -3,7 +3,7 @@
     <VersionMajor>2</VersionMajor>
     <VersionMinor>2</VersionMinor>
     <VersionPatch>207</VersionPatch>
-    <ReleaseSuffix Condition=" '$(ReleaseSuffix)' == '' ">preview</ReleaseSuffix>
+    <ReleaseSuffix Condition=" '$(ReleaseSuffix)' == '' ">rtm</ReleaseSuffix>
 
     <MajorMinorVersion>$(VersionMajor).$(VersionMinor)</MajorMinorVersion>
     <CliVersionNoSuffix>$(VersionMajor).$(VersionMinor).$(VersionPatch)</CliVersionNoSuffix>
```